registration/registration-services/src/main/java/io/mosip/registration/service/external/impl/PreRegZipHandlingServiceImpl.java

```diff
@@ -262,6 +262,7 @@ private void parseDemographicJson(BufferedReader bufferedReader, ZipEntry zipEnt
 							documentDto.setFormat(fieldValue.getString("format"));
 							documentDto.setType(fieldValue.getString("type"));
 							documentDto.setValue(fieldValue.getString("value"));
+							documentDto.setRefNumber((fieldValue.getString("docRefId")));
```

Comment:
what if docRefId is null in the pre-reg packet ?

Reply:
Resolved